src/common/feedGenerator.ts

```diff
@@ -307,7 +308,7 @@ const sourcesToFilters = ({
 };
 
 export const feedToFilters = async (
-  con: DataSource,
+  con: DataSource | EntityManager,
```

Comment:
Main change for digest to work, in digest we inject `queryRunner.manager` from `queryReadReplica`.